lib/src/main/res/values/strings.xml

```diff
@@ -8,7 +8,7 @@
     <string name="voice_placeholder">Say a what3words address</string>
     <string name="voice_try_again">Please try again</string>
     <string name="voice_error_label">Don’t say street addresses, we only accept what3words addresses.</string>
-    <string name="near">near %s</string>
+    <string name="near_placeholder">near %s</string>
```

Comment:
I think it is used for showing more information below the three-word text which is not a placeholder. Regarding the issue, it seems to be caused by the conflict in resource ID from multiple libraries so I think that might happen if we do not make it unique.

In my view, we can name it as the naming convention meeting which is finalized while @mani-w3w is in the Viet Nam office.